packages/shared/src/components/squads/PublicSquadEligibilityCard.tsx

```diff
@@ -48,7 +46,7 @@ const PublicSquadEligibilityCard = ({
   const isEligible = postsCount >= MIN_POSTS;
   const effectivePostsCount = isEligible ? MIN_POSTS : postsCount;
 
-  const CardComponent = shouldUseMobileFeedLayout ? CardV1 : Card;
```

Comment:
Just leftover changes from mobile cleanup, TS was complaining.

Reply:
`shouldUseMobileFeedLayout` no longer exists

Reply:
Ah true, not sure how this got past without fixing when I merged the layout PR, but it definitely needs to be replaced with `shouldUseListFeedLayout`.

I'll raise a PR for this (https://github.com/dailydotdev/apps/pull/3127), please remove your changes 🙏

Reply:
🫡